packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -220,6 +223,60 @@ describe("IModelTransformer", () => {
     targetDb.close();
   });
 
+  it("should test rangesFromRangeAndSkipped", async () => {
```

Comment:
can you put this in a separate test file?
also ideally each of the comments would be a separate `it(() => {})`

Reply:
done